.github/workflows/cs-pr-what-if.yml

```diff
@@ -0,0 +1,40 @@
+---
+name: CS PR Check Environment Bicep What-If
+env:
+  DEPLOY_ENV: cs-pr
+  DEPLOYMENT_NAME_SUFFIX: "-dev-${GITHUB_RUN_ID}"
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'dev-infrastructure/**/*.bicep'
+      - 'dev-infrastructure/**/*.bicepparam'
+      - 'dev-infrastructure/Makefile'
+      - '.github/workflows/cs-pr-bicep-what-if.yml'
```

Comment:
Same question above.  Do we need to watch / check on config?

Reply:
good catch. fixed